src/HTTP.jl

```diff
@@ -485,6 +494,7 @@ include("ConnectionRequest.jl");        using .ConnectionRequest
 include("DebugRequest.jl");             using .DebugRequest
 include("StreamRequest.jl");            using .StreamRequest
 include("ContentTypeRequest.jl");       using .ContentTypeDetection
+include("exceptions.jl")
 
 """
```

Comment:
After review is done, 
can we move this whole sectuion to a new file
like 
`strack.jl`  or something?

Reply:
Do you find it

di`strack`ting?

😎